serving/src/main/resources/application.conf

```diff
@@ -73,6 +73,14 @@ defaults {
     ssl-verify = false
     ssl-verify = ${?SSL_VERIFY_CONFIG}
     # SSL Cert Configs for Git
+
+    #S3 settings
+    s3-download-timeout = 300 seconds
+    s3-download-timeout = ${?S3_DOWNLOAD_TIMEOUT_CONFIG}
+    s3-access-key-appender = "_AWS_ACCESS_KEY_ID"
```

Comment:
The name "appender" makes this sound like this actually does something (it's a verb), for example like downloader. If this is simply meant to be a postfix then let's call it that please.